CMakeLists.txt

```diff
@@ -429,6 +429,15 @@ target_include_directories(local_install INTERFACE ${CMAKE_SOURCE_DIR})
 # Enable testing
 enable_testing()
 
+if(TILEDB_TESTS)
+  # Add custom Catch2 entry point that suppresses message boxes on debug assertion
+  # failures on Windows CI.
+  find_package(Catch2 REQUIRED)
+  add_library(tdb_Catch2WithMain STATIC
```

Comment:
Perhaps we can name it TileDB::Catch2WithMain or similar, to not use underscores as namespace specifier

Reply:
Only imported and alias targets can have double colons. And we have not defined any other `TileDB::` targets for internal use.

Reply:
Yea, but we can alias this target. I usually have add_library and alias right next to each other. But as I said, feel free to ignore this

Reply:
Renamed target to `tiledb_Catch2WithMain`.